src/client/dialog-demo-bootstrap/styles.css

```diff
@@ -1,6 +1,10 @@
 /* needed to make consistent test snapshots across OSs */
+html {
+  height: 100%;
+}
 body {
-  font-family: Arial !important;
+  font-family: Roboto !important;
```

Comment:
I think loading an external font is more consistent across operating systems than using a default like Arial.